### PR TITLE
Update routes_test.rb

### PR DESCRIPTION
**Pull request by cmcgee0308**
Fixes #(update with issue number)

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @cmcgee0308, can you add a link to the issue? 

> One thing that can help to get started is to make sure you've included a link back to the original issue you're solving, in the
> format fixes #0000 (for example)

**Comment by cmcgee0308** [comment]
#858 Sorry about that! I also see that there seems to be an error? How do I go about fixing this?

File: test/integration/routes_test.rb [hunk 1]
```diff
@@ -112,5 +112,9 @@ class RoutesTest < ActionDispatch::IntegrationTest
   test "test get request for sets calibrated" do
     assert_routing({ path: '/sets/calibrated', method: 'get' }, {controller: 'sets', action: 'show', id: 'calibrated'})
   end
+	
+  test "test get request for spectrums anonymous" do
+    assert_routing({ path: '/spectrums/anonymous', method: 'get' }, {controller: 'spectrums', action: 'anonymous'})
```
Comment (TildaDares) [inline test/integration/routes_test.rb:117]: @cmcgee0308 You can fix it by updating this line to this
```suggestion
    assert_routing({ path: '/spectrums/anonymous', method: 'get' }, {controller: 'spectrums', action: 'show', id: 'anonymous'})
```

Comment (cmcgee0308) [reply test/integration/routes_test.rb]: Okay I made the changes!

**Review by TildaDares — APPROVED** [review]
Thanks @cmcgee0308. Great job!

**Comment by cmcgee0308** [comment]
Thank you @TildaDares ! Is it okay to ask what exactly this issue does or what the lines I put in do? I've never worked with code like this before.

**Comment by TildaDares** [comment]
@cmcgee0308 The code you’ve just added is a routes test. It basically tests that a route maps to the correct controller action.

**Comment by cmcgee0308** [comment]
Thank you @TildaDares I appreciate it!

**Pull request merged**